extension/docs/tdigest.md

```diff
@@ -130,23 +206,66 @@ This will construct and return a TDigest with the specified number of buckets ov
 For this examples assume we have a table 'samples' with a column 'weights' holding `DOUBLE PRECISION` values.  The following will simply return a digest over that column
 
 ```SQL ,ignore
-SELECT timescale_analytics_experimental.tdigest(100, data) FROM samples;
+SELECT tdigest(100, data) FROM samples;
 ```
 
 It may be more useful to build a view from the aggregate that we can later pass to other tdigest functions.
 
 ```SQL ,ignore
 CREATE VIEW digest AS
-    SELECT timescale_analytics_experimental.tdigest(100, data)
+    SELECT tdigest(100, data)
     FROM samples;
 ```
 
 ---
 
-## **tdigest_min** <a id="tdigest_min"></a>
+## **tdigest (summary form)** <a id="tdigest-summary"></a>
+```SQL ,ignore
+tdigest(
+    digest TDigest
+) RETURNS TDigest
+```
+
+This will combine multiple already constructed TDigests, if they were created with the same size. This is very useful for re-aggregating digests already constructed using the [point form](#tdigest).
```

Comment:
I think we should probably add a note about how this potentially introduces more error and the non-deterministic nature of this sketch here?

Reply:
ie: Note that the tdigest approximation will not necessarily produce the same results with different ordered inputs, nor will it necessarily produce the same results when used with re-aggregation steps as if it were computed using the point form. Usually the differences will be minor, especially if there are relatively large numbers of values in each of the digests before re-aggregating.